projects/plugins/beta/src/class-clicommand.php

```diff
@@ -146,6 +147,13 @@ public function activate( $args ) {
 			$premsg = __( 'Activating %1$s master branch', 'jetpack-beta' );
 			// translators: %1$s: Plugin name.
 			$postmsg = __( '%1$s is now on the master branch', 'jetpack-beta' );
+		} elseif ( 'trunk' === $args[1] ) {
```

Comment:
So again, with the "little work" we can delete the "master" case above and let this be
```php
		if ( 'trunk' === $args[1] || 'master' === $args[1] ) {
```